Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Use an official Python runtime as a parent image
+FROM python:3.8
+MAINTAINER label="Rohan Rustagi"
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy your Python script into the container
+COPY *.py .
```

Comment:
This wouldn't scale well with projects as they grow, as it indiscriminately copies all files from the build context into the docker image